elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -233,17 +247,18 @@ def render(element_html, data):
         solution_header = pl.get_string_attrib(element, 'solution-header', SOLUTION_HEADER_DEFAULT)
 
         mcq_options = data['params'][answer_name]
-        mcq_options = filter_multiple_from_array(mcq_options, ['inner_html', 'uuid'])
+        mcq_options = filter_multiple_from_array(mcq_options, ['inner_html', 'uuid', 'distractor-feedback'])
 
         if answer_name in data['submitted_answers']:
             student_previous_submission = filter_multiple_from_array(data['submitted_answers'][answer_name], ['inner_html', 'uuid', 'indent'])
-            mcq_options = [opt for opt in mcq_options if opt not in filter_multiple_from_array(student_previous_submission, ['inner_html', 'uuid'])]
+            mcq_options = [opt for opt in mcq_options if opt['uuid'] not in {block['uuid'] for block in student_previous_submission}]
 
```

Comment:
@CoffeePoweredComputers the change on this line should fix the bug you had with blocks being duplicated. The blocks were failing the set membership check here because of the new 'distractor-info' property you added. Obviously, this was a poorly designed check, so I changed it to just check UUIDs instead (as we should). 

I can't test the fix myself though, because I'm getting errors other places in the code, you must have fixed them locally but not pushed yet. Anyway, let me know how it goes!

Reply:
I figured out what was causing the crash (the problem that I was using it on has feedback=none, so the 'badge_type' was never set), so I was able to test. 

It works!